docs/BUILD.bazel

```diff
@@ -127,12 +127,14 @@ PAGES = dict([
     page(
         name = "rust_repositories",
         symbols = [
+            "rules_rust_depednencies",
```

Comment:
Drive-by comment, but should it be `rules_rust_dependencies` here?

Reply:
Oops! Thanks! https://github.com/bazelbuild/rules_rust/pull/1110